guides/common/modules/con_host-group-settings-and-nested-host-groups.adoc

```diff
@@ -0,0 +1,111 @@
+
+.Nested Host Group Hierarchy
+====
+You create a top-level host group named `Base` and two nested host groups named `Webserver` and `Storage`.
+The nested host groups are associated with multiple hosts.
+You also create host `custom.example.com` that is not associated with any host group.
+
+[cols="1,1,1"]
+|===
+|Top-level host group |Nested host group |Host
+
+.5+|*Base*
+
+This host group applies the `{EL}{nbsp}8.8` operating system setting.
+
+.2+|*Webserver*
+
+This host group applies the `linux-system-roles.selinux` Ansible role.
+
+|*webserver1.example.com*
+|*webserver2.example.com*
+
+.2+|*Storage*
+
+This host group applies the `linux-system-roles.postfix` Ansible role.
+
+|*storage1.example.com*
+|*storage2.example.com*
+
+|
+|*custom.example.com*
+
+|===
+
+In this hierarchy:
+
+* `Base` defines the settings common to all hosts. All hosts use {EL}{nbsp}8.8 as their operating system.
+* `webserver1.example.com` and `webserver2.example.com` use the `linux-system-roles.selinux` Ansible role.
+`storage1.example.com` and `storage2.example.com` use the `linux-system-roles.postfix` Ansible role.
+* `custom.example.com` inherits none of the Ansible roles defined by the host groups.
+====
+
+{Project} applies host settings in the following order when nesting host groups:
+
+* Host settings take priority over host group settings.
+* Nested host group settings take priority over parent host group settings.
+
+.Nested Host Group Settings
+====
+You define different values for the operating system and Ansible role settings on the top-level host group (*Base*) and nested host groups (*Webserver* and *Storage*).
+
+[cols="1,1,1"]
+|===
+|Top-level host group |Nested host group |Host
+
+.3+a|*Base*
+
+This host group applies these settings:
+
+* The `{EL}{nbsp}8.8` operating system
+* The `linux-system-roles.selinux` Ansible role
+
+a|*Webserver*
+
+This host group applies these settings:
+
+* The `{EL}{nbsp}8.9` operating system
+* No Ansible role
+
+a|*webserver1.example.com*
+
+*webserver2.example.com*
+
+a|*Storage*
+
+This host group applies these settings:
+
+* No operating system
+* The `linux-system-roles.postfix` Ansible role
+
+a|*storage1.example.com*
+
+*storage2.example.com*
+
+|
+a|*custom.example.com*
+
+|===
+
+{Project} applies settings from nested groups in the following way:
+
+* *webserver1.example.com* and *webserver2.example.com* inherit these settings:
+** The `{EL}{nbsp}8.9` operating system from host group *Webservers*
+** The `linux-system-roles.selinux` Ansible role from host group *Base*
+* *storage1.example.com* and *storage2.example.com* inherit these settings:
+** The `{EL}{nbsp}8.8` operating system from host group *Base*
+** The `linux-system-roles.postfix` Ansible role from host group *Storage*
+* *custom.example.com* inherits this setting:
+** The `{EL}{nbsp}8.8` operating system from host group *Base*
+
```

Comment:
Why not having these explanations in the last column inside the table? Similarly for the other example.

Reply:
I thought about this, and I must say I like the explanations where there are now. If I added them to the table, the table might get too information heavy. As always, let me know if you see it differently :)

Reply:
I don't think it's necessarily a bad thing. When I was going through the table, I was expecting the "results of the maths" in the last column and I was surprised to find them under the tables.
Like this, the information is spread over too much area. Also, consider how we display tables on the CP: The user can enlarge the table and then they would to exit the table to read the rest of the information.
But I'm leaving the final decision to you.

Reply:
Thanks for explaining @Lennonka. I gave it a try (have I mentioned how I hate tables in asciidoc..?). Can you take another look?